### PR TITLE
feature: Add .NET 5 support

### PR DESCRIPTION
**Pull request by glennawatson**

### CODE REVIEW

**Comment by glennawatson** [comment]
The lower code coverage is correct, since its taking into account a lot more projects now.

File: src/Splat.DryIoc.Tests/Splat.DryIoc.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
```
Comment (RLittlesII) [inline src/Splat.DryIoc.Tests/Splat.DryIoc.Tests.csproj:1]: Microsoft.NET.Sdk

**Review by RLittlesII — APPROVED** [review]
some of the sdks might need to be changed.

File: src/Splat.SimpleInjector.Tests/Splat.SimpleInjector.Tests.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
```
Comment (RLittlesII) [inline src/Splat.SimpleInjector.Tests/Splat.SimpleInjector.Tests.csproj:1]: Microsoft.NET.Sdk

File: src/Splat.DryIoc.Tests/Splat.DryIoc.Tests.csproj (hunk 1 above)
Comment (RLittlesII) [inline src/Splat.DryIoc.Tests/Splat.DryIoc.Tests.csproj:1]:
```suggestion
﻿<Project Sdk="Microsoft.NET.Sdk">
```

File: src/Splat.SimpleInjector.Tests/Splat.SimpleInjector.Tests.csproj (hunk 2 above)
Comment (RLittlesII) [inline src/Splat.SimpleInjector.Tests/Splat.SimpleInjector.Tests.csproj:1]:
```suggestion
﻿<Project Sdk="Microsoft.NET.Sdk">
```

**Pull request merged**